jenkins/image_building/build-image.sh

```diff
@@ -20,29 +20,18 @@ export IMAGE_OS="${IMAGE_OS}"
 export IMAGE_TYPE="${IMAGE_TYPE}"
 
 # shellcheck disable=SC1091
-source "${current_dir}/upload-ci-image.sh"
+. "${current_dir}/upload-ci-image.sh"
 
 # Disable needrestart interactive mode
 sudo sed -i "s/^#\$nrconf{restart} = 'i';/\$nrconf{restart} = 'a';/" /etc/needrestart/needrestart.conf > /dev/null || true
 
 sudo apt-get update
-sudo apt-get install python3-pip qemu qemu-kvm -y
-sudo pip3 install diskimage-builder python-openstackclient
```

Comment:
We don't need python-openstackclient anymore?

Reply:
No. We don't interact with openstack or ironic. Only diskimage-builder is needed.